### ISSUE TITLE
StringType documentation has incorrect min_length

### ISSUE THREAD

**Issue by di** [schematics/schematics#385]
From http://schematics.readthedocs.org/en/latest/api/types.html#schematics.types.base.StringType:

> A unicode string field. Default minimum length is one. If you want to accept empty strings, init with min_length 0.

However, the following does not raise an exception:

```python
from schematics.types import StringType
st = StringType()
st.validate('')
```

Seems like the documentation never got updated as mentioned in #151.

**Comment by bintoro** [comment]
The docstring was fixed in 690b5bdb854af489f01b3f724ad8c5d56ca04a0c, and I'm seeing the correct docstring on Read the Docs as well. Might you be reading a cached version?

**Comment by di** [comment]
Sorry, yes, that's the case.
